Finjector.Web/ClientApp/src/components/Entry/FolderSearch.tsx

```diff
@@ -42,6 +42,11 @@ const FolderSearch = ({
         setSelectedFolder(
           data.find((f) => f.teamName === "Personal" && f.name === "Default")
         );
+        //we also want to set the folderid here
+        updateFolderId(
+          data.find((f) => f.teamName === "Personal" && f.name === "Default")
+            ?.id ?? 0
+        );
       }
     }
   }, [data, initialFolderId]);
```

Comment:
@srkirkland @laholstege If I add the update folder here, but don't include it in the array directly above, it works.
If I add it to the array, you can't clear it out and select a different folder.

Reply:
@jSylvestre it makes more sense to me to do this inside of Entry's useEffect(), like: 
```
  // if we load up new data, update the chart
  useEffect(() => {
    if (savedChartQuery.data) {
      const { chart } = savedChartQuery.data;
      if (!chart.canEdit) {
        chart.folderId = 0;
      }
```
what do you think @srkirkland

Reply:
https://github.com/ucdavis/finjector/pull/248